src/blueprint/blueprint.rs

```diff
@@ -86,6 +86,7 @@ pub struct ObjectTypeDefinition {
   pub fields: Vec<FieldDefinition>,
   pub description: Option<String>,
   pub implements: BTreeSet<String>,
+  pub cache: Option<Cache>,
```

Comment:
Create a `blueprint::Cache`. 
- Use it **only on fields** and not on types.
- Cache should also have a default hasher inside it.

```rust

#[derive(Clone, Debug)]
pub struct Cache {
  pub max_age: NonZeroU64,
  pub hasher: DefaultHasher,
}
```